domains/runtime/core-payments/src/runtime.rs

```diff
@@ -308,8 +308,8 @@ construct_runtime!(
         TransactionPayment: pallet_transaction_payment = 3,
 
         // messenger stuff
-        Messenger: pallet_messenger = 7,
-        Transporter: pallet_transporter = 8,
+        Messenger: pallet_messenger = 6,
```

Comment:
Do this mean the messenger pallet indexes of all the domain runtimes should be the same? It's valuable to add a comment.

Reply:
yes. Just the Messenger index so that we can decode the extrinsinc on other runtimes as needed